index.html

```diff
@@ -1,133 +1,167 @@
+						<li class="social">Log Out</li>
+					</ul>
+				</div>
+				<nav id="nav-bar">
+					<a href="index.html">
+						<div id="logo-container">
+							<img src="Images/torii (1).png" alt="logo" class="logo" />
+							<div id="logo-subContainer">
+								<h1>Japanese</h1>
+								<h3>Culture Festival 2023</h3>
+							</div>
+						</div>
+					</a>
+					<ul id="nav-list">
+						<li class="nav-sections"><a href="About.html">About</a></li>
+						<li class="nav-sections">Program</li>
+						<li class="nav-sections">Join</li>
+						<li class="nav-sections">Sponsor</li>
+						<li class="nav-sections">News</li>
+						<li class="nav-sections">Campaign</li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main>
+			<section id="title">
+				<div>
+					<h2>こんにちは世界</h2>
+					<h1>Japanese Culture Festival 2023</h1>
+				</div>
+				<div>
+					<p
+						>A joyful celebration believing in the value of openess and sharing, creating a positive
+						change with people from over 80 countries is taking place in Tokyo, Japan.</p
+					>
+					<h1>2023.06.15(THU) ~ 16(FRI)</h1>
+					<p>@National Gobernment of Japan, Art Center Nabi and more</p>
+				</div>
+			</section>
+			<section id="program">
+				<div class="title-div">
+					<h2>Main Program</h2>
+					<hr class="hr-bar">
+				</div>
+				<div id="programs-container">
+					<div class="programs">
+						<img src="Images/program_icon_01.png" alt="lecture" />
+						<p class="title">Lecture</p>
+						<p>Listen to the presentes about the messages of sharing and opening.</p>
+					</div>
+					<div class="programs">
+						<img src="Images/program_icon_02.png" alt="exhibition" />
+						<p class="title">CC Exhibition</p>
+						<p
+							>Appreciate various creations applying CC license of artists, organized from Art
+							Center Nabi.</p
+						>
+					</div>
+					<div class="programs">
+						<img src="Images/program_icon_03.png" alt="forum" />
+						<p class="title">Forum</p>
+						<p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+					</div>
+					<div class="programs">
+						<img src="Images/program_icon_04.png" alt="eat" />
+						<p class="title">Eat</p>
+						<p>Taste food you'd never try and delight yourself with the asian gastronomy.</p>
+					</div>
+					<div class="programs">
+						<img src="Images/program_icon_05.png" alt="landscape" />
+						<p class="title">Landscape</p>
+						<p>Appreciate the beautiful view of the capital with blooming cherry blossoms.</p>
+					</div>
+					<button type="button">Join Japanese Festival 2023</button>
+				</div>
+				<p id="hiden-program">SEE THE HOLE PROGRAM</p>
+			</section>
+			<section id="main-activities">
+				<div class="title-div">
+					<h2>Main Activities</h2>
+					<hr class="hr-bar">
+				</div>
+				<div id="activities-container"> </div>
+			</section>
+			<section id="partners" class="hide">
+				<div class="title-div">
+					<h1>Partner</h1>
```

Comment:
According to the design given in the mobile view, there is a `MORE`  button that selects the remaining activities listed, however this feature was not implemented in your design. Could you kindly add this feature to have a better UI 😄 

Your design :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/c1839e80-5fb3-4b5e-b7d4-d8e08e625f20)

Design given :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/0b3bc5e1-8307-4cb4-b9ca-1c37b09dae61)